### ISSUE TITLE
tag name into manifest when gb vendor fetch -tag <tag_name>

### ISSUE THREAD

**Issue by jerome-laforge** [constabulary/gb#636]
Hello,
Is it possible to add the tag name into manifest when we fetch the vendor by tag name ?
gb vendor fetch -tag 1.0.0  somewhere

have this 
```json
{
	"importpath": "somewhere",
	"repository": "https://somewhere",
	"tag": "1.0.0",
	"branch": "refs/tags/"
}
```

instead of 
```json
{
	"importpath": "somewhere",
	"repository": "https://somewhere",
	"tag": "a83829b6f1293c91addabc89d0571c246397bbf4",
	"branch": "HEAD"
}
```

Thx again for your kindness and your patience
Jérôme

**Comment by davecheney** [comment]
Gb vendor already support this, however it does not record the tag name,
but the matching revision. This was an oversight in the v0 manifest format.
Rectifying it will require bumping the manifest format which is a large
piece of work.

**Comment by jerome-laforge** [comment]
Maybe instead of modify the manifest format, is it possible to add the tag
name as comment?
That allows quickly know on which tag the dependency is pinned without
access to history of VCS.

Le 14 juil. 2016 10:23 PM, "Dave Cheney" <notifications@github.com> a
écrit :

**Comment by davecheney** [comment]
Nope, sorry. The manifest format is versioned to permit this kind of
change, there is no need for a workaround.
